--- a/src/buildtools/geninfocontrib_h.sh
+++ b/src/buildtools/geninfocontrib_h.sh
@@ -299,19 +299,19 @@ if test x"$1" = "xinfocontrib.h"; then

   $ECHO ""
   $ECHO "vice_team_t core_team[] = {"
-  cat coreteam.tmp | sed "s/__VICE_CURRENT_YEAR__/$year/g"
+  test -f coreteam.tmp && cat coreteam.tmp | sed "s/__VICE_CURRENT_YEAR__/$year/g"
   rm -f coreteam.tmp
   $ECHO "    { NULL, NULL, NULL }"
   $ECHO "};"
   $ECHO ""
   $ECHO "vice_team_t ex_team[] = {"
-  cat exteam.tmp | sed "s/__VICE_CURRENT_YEAR__/$year/g"
+  test -f exteam.tmp && cat exteam.tmp | sed "s/__VICE_CURRENT_YEAR__/$year/g"
   rm -f exteam.tmp
   $ECHO "    { NULL, NULL, NULL }"
   $ECHO "};"
   $ECHO ""
   $ECHO "char *doc_team[] = {"
-  cat docteam.tmp
+  test -f docteam.tmp && cat docteam.tmp
   rm -f docteam.tmp | sed "s/__VICE_CURRENT_YEAR__/$year/g"
   $ECHO "    NULL"
   $ECHO "};"
@@ -318,7 +318,7 @@ if test x"$1" = "xinfocontrib.h"; then
   $ECHO ""
   $ECHO "vice_trans_t trans_team[] = {"
-  cat transteam.tmp | sed "s/__VICE_CURRENT_YEAR__/$year/g"
+  test -f transteam.tmp && cat transteam.tmp | sed "s/__VICE_CURRENT_YEAR__/$year/g"
   rm -f transteam.tmp
   $ECHO "    { NULL, NULL, NULL, NULL }"
   $ECHO "};"
